src/main/java/edu/cornell/kfs/concur/service/impl/ConcurEventNotificationConversionServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package edu.cornell.kfs.concur.service.impl;
+
+import java.text.ParseException;
+
+import org.kuali.rice.core.api.datetime.DateTimeService;
+
+import edu.cornell.kfs.concur.businessobjects.ConcurEventNotification;
+import edu.cornell.kfs.concur.rest.xmlObjects.ConcurEventNotificationDTO;
+import edu.cornell.kfs.concur.service.ConcurEventNotificationConversionService;
+
+public class ConcurEventNotificationConversionServiceImpl implements ConcurEventNotificationConversionService {
+    
+    protected DateTimeService dateTimeService;
+
+    @Override
+    public ConcurEventNotification convertConcurEventNotification(ConcurEventNotificationDTO concurEventNotificationDTO) {
+        ConcurEventNotification concurEventNotification = new ConcurEventNotification();
+        concurEventNotification.setContext(concurEventNotificationDTO.getContext());
+        try {
+            concurEventNotification.setEventDateTime(dateTimeService.convertToSqlDate(concurEventNotificationDTO.getEventDateTime()));
+        } catch (ParseException e) {
+            concurEventNotification.setEventDateTime(dateTimeService.getCurrentSqlDate());
```

Comment:
This event date time will be used to fail requests that could not be processed on our side in 48 hours (still need to create a story for this, maybe we an talk about this after the break). That is the time the time the event notification was generated. It seemed having it defaulted to current time rather than failing the whole thing would make more sense. But maybe I should send back a failure response in this case?